packages/saslprep/src/generate-code-points.ts

```diff
@@ -1,13 +1,6 @@
 import { gzipSync } from 'zlib';
 import bitfield from 'sparse-bitfield';
 import * as codePoints from './code-points-src';
-import { createWriteStream } from 'fs';
-
-if (!process.env.GENERATE_CODE_POINTS) {
-  // TODO(COMPASS-7555): fix codepoint generation=
-  process.exitCode = 0;
-  process.exit();
-}
```

Comment:
I don't think we can easily remove the `createWriteStream` addition again because now this script has 2 output files

But we could also just remove this conditional as a revert, I think